pkg/ui/workspaces/cluster-ui/src/insights/utils.ts

```diff
@@ -460,7 +460,7 @@ export function getStmtInsightRecommendations(
     indexRecommendations: insightDetails.indexRecommendations,
     databaseName: insightDetails.databaseName,
     elapsedTimeMillis: insightDetails.elapsedTimeMillis,
-    contentionTime: insightDetails.totalContentionTime.asMilliseconds(),
+    contentionTime: insightDetails.totalContentionTime?.asMilliseconds(),
```

Comment:
this was just fixed on a PR just merged: https://github.com/cockroachdb/cockroach/pull/92571/files#diff-3805bc3aeac419ce79c8b16f1b6dd247608f9a30ec3bfa3e6007cb3350b8eeb7R463

Reply:
Nice, I'll remove that from the commit description!